src/app.js

```diff
@@ -0,0 +1,51 @@
+// const app = () => {
+//   const apiRequest = new XMLHttpRequest();
+//   const parent = document.getElementById('container');
+//   parent.addEventListener('click', (e) => {
+//     if (e.target.classList.contains('submit_button')) {
+//       e.preventDefault();
+//       const cityInput = document.getElementById('city');
+//       const chosenCity = cityInput.value;
```

Comment:
- [ ] Since we are about to push this code to production, we should delete all commented code.